pkg/controller/operatorhub/operatorhub_controller.go

```diff
@@ -49,16 +49,10 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// We only care if the event came from the cluster config.
 	pred := predicate.Funcs{
 		CreateFunc: func(e event.CreateEvent) bool {
-			if e.Meta.GetName() == operatorhub.DefaultName {
-				return true
-			}
-			return false
+			return e.Meta.GetName() == operatorhub.DefaultName
```

Comment:
This applies to the other changes in this package, but just simplifying the logic for these predicate functions to avoid having to make uneeded additional comparisons.